ui.R

```diff
@@ -63,7 +65,7 @@ body <- bs4DashBody(
       bs4TabItem('sobre',
                  tagList(
                    # asynchronously register the map of Rondonia _once_
-                   e_map_register_ui("Rondonia", "https://raw.githubusercontent.com/luizpedone/municipal-brazilian-geodata/master/data/RO.json", async = TRUE),
+                   e_map_register_ui("Rondonia", "www/rondonia.json", async = TRUE),
                    tags$img(src = 'home-banner.jpg', height = 200, width = 1040, align = 'center'),
                    
                    #bs4Box(

```

Comment:
@JohnCoene 
I do exactly what you recommend on the code, and only returns the blank map.

![Sem título](https://user-images.githubusercontent.com/31769064/82849518-6006e800-9ec6-11ea-8a1c-2f1a945a878c.png)